exporter/jaegerexporter/exporter.go

```diff
@@ -100,3 +135,54 @@ func (s *protoGRPCSender) pushTraceData(
 
 	return 0, nil
 }
+
+func (s *protoGRPCSender) shutdown(context.Context) error {
+	close(s.stopCh)
+	s.stopWg.Wait()
+	view.Unregister(MetricViews()...)
+	return nil
+}
+
+func (s *protoGRPCSender) start(context.Context, component.Host) error {
+	view.Register(MetricViews()...)
```

Comment:
Until a better solution move the views registration to the service https://github.com/open-telemetry/opentelemetry-collector/blob/master/service/telemetry.go#L65, I will create a mechanism for registration later.

Reply:
Alright, moved there (and sorted the lines there).